packages/cli/src/Server.ts

```diff
@@ -964,10 +964,10 @@ class App {
 					}
 				}
 
-				// required due to atomic update
-				updateData.updatedAt = this.getCurrentDate();
-
-				await validateEntity(updateData);
+				if (Object.keys(updateData).length > 1) {
```

Comment:
Hum what bothers me in this case is that I can use a payload such as `{"name": "Invalid name"}` and it would pass 😅 and considering in the future we might add other validations, I think this could be bad.

Maybe use something like `Object.keys(updateData).length > 1 || updateData.active === undefined` is better WDYT?

Reply:
Mistakenly assumed this endpoint receives either the entire workflow or this small payload, but clearly if it is PATCH then it will allow for partial updates. Will fix.